pages/index.vue

```diff
@@ -46,14 +36,22 @@ definePageMeta({
     </div>
 </template>
 
+<script setup>
+const { locale } = useI18n()
+const localePath = useLocalePath()
+definePageMeta({
+    title: 'Haroohie Translation Club',
+    description: 'Welcome to the Haroohie Translation Club! We\'re dedicated to translating games from the Haruhi Suzumiya series into English!',
```

Comment:
hmm we should look at pulling these strings from the locale strings if possible, probably. I don't think I did that last time but it would be good.